CMakeLists.txt

```diff
@@ -615,4 +627,29 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
+
+# The prebuilt IDL headers aren't suitable for 32 bit builds, attempt to use the ones 
+# generated when building corguids by including them before the prebuilt ones
+if (WIN32 AND CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
I'm not a fun of such combined variables, we get more variables for unclear benefit. 

Besides, I hope that this particular section will go away or be improved sooner or later. It works but it's kind of ugly. If we find that this issue affects other 32 bit platforms we'll have to commit prebuilt headers for 32 bit anyway.